data/announcements/scheduled.yaml

```
@@ -103,3 +103,14 @@ announcements:
   message: |
     2 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     December 9 + 10, 2021.
+
+- name: Kubecon 2022 EU
+  startTime: 2022-05-09T00:00:00  #Added in https://github.com/kubernetes/website/pull/33562
+  endTime: 2022-05-20T18:00:00
+  style: "background: linear-gradient(90deg, rgb(253, 225, 203) 0%, rgb(236, 107, 67) 100%)"
```

I would use black text, not white. The white text is too low contrast against the background.

Also, move the blend point. Here's how that might look:
```suggestion
  style: >-
    background: linear-gradient(90deg, rgb(253, 225, 203) 0%, rgb(236, 107, 67) 75%, rgb(236, 107, 67) 100%);
    color: #000;
```